src/TTextEdit.cpp

```diff
@@ -1958,7 +1958,7 @@ inline QString TTextEdit::convertWhitespaceToVisual(const QChar& first, const QC
         case 0x206D:                    return htmlCenter(tr("{activate arabic form-shaping}", "Unicode U+206D codepoint.")); break;
         case 0x206E:                    return htmlCenter(tr("{national digit shapes}", "Unicode U+206E codepoint.")); break;
         case 0x206F:                    return htmlCenter(tr("{nominal Digit shapes}", "Unicode U+206F codepoint.")); break;
-        case 0x3000:                    return htmlCenter(tr("{ideaographic space}", "Unicode U+3000 codepoint - ideaographic (CJK Wide) space")); break;
+        case 0x3000:                    return htmlCenter(tr("{ideographic space}", "Unicode U+3000 codepoint - ideographic (CJK Wide) space")); break;
```

Comment:
Are you sure? I can find evidence of the spelling I used, for instance on: https://quizlet.com/201728184/developing-chinese-civilization-flash-cards/ . OTOH The Unicode listing does use the spelling you are suggesting e.g. https://unicode-table.com/en/3000/ ...

:man_shrugging:

Reply:
I went by a unicode list like that one, plus the counts when I searched for ideaographic vs ideographic and ideaogram vs ideogram.